workflows/argo/training.yaml

```diff
@@ -56,12 +56,18 @@ spec:
             echo "Validation Data Configuration:"
             cat validation_data.yaml
 
+            if grep -q '[^[:space:]]' "validation_data.yaml";
```

Comment:
Is this tested? What would happen if user passes some string other than `" "` for this argument?

Reply:
The same would happen as for if a bad train or test data config was passed- the `validation-data-config` arg would be passed to the python call and then some error would be raised when converting it to a DataConfig object. I tested the if statement in a simple test case of passing a "config" through the command line and verifying that it either did not add a validation arg for an empty string passed, or added the validation arg if a string was passed.
```
#!/bin/bash
validation_config=$1


cat <<EOF >validation.yaml
$validation_config
EOF

if grep -q '[^[:space:]]' "validation.yaml";
then
     validation_arg="--validation-data-config validation.yaml"
else
     validation_arg=" "
fi

echo "validation arg: $validation_arg"
```